certora/specs/EnumerableMap.spec

```diff
@@ -48,7 +48,7 @@ invariant indexedContained(uint256 index)
     {
         preserved {
             requireInvariant consistencyIndex(index);
-            requireInvariant consistencyIndex(to_uint256(length() - 1));
+            requireInvariant consistencyIndex(require_uint256(length() - 1));
```

Comment:
I'd do `assert_uint256` just to be sure. `length()` should never return anything bigger than 2**129-1.